### PR TITLE
Output data for Pod status and phase

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @bbguimaraes

### CODE REVIEW

File: hack/monitor_prow.py [hunk 1]
```diff
@@ -67,11 +67,14 @@ def highlight(log_dir, dc):
...
-                    log_lines.append(YELLOW + l + RESET)
-                elif error in l or fatal in l:
-                    log_lines.append(RED + l + RESET)
+            try:
+                for l in exec_cmd(*cmd).splitlines():
+                    if warn in l:
+                        log_lines.append(YELLOW + l + RESET)
+                    elif error in l or fatal in l:
+                        log_lines.append(RED + l + RESET)
+            except:
```
Comment (bbguimaraes) [inline hack/monitor_prow.py]:
```suggestion
            except subprocess.CalledProcessError:
```

so we don't catch other potential errors.

https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError

Comment (bbguimaraes) [reply hack/monitor_prow.py]: Oh, pylint won.

Comment (petr-muller) [reply hack/monitor_prow.py]: pylint is da best!

**Comment by stevekuznetsov** [comment]
@bbguimaraes done

**Pull request merged**